src/main.cpp

```diff
@@ -67,6 +71,13 @@ int main(int argc, char* argv[])
 
   const auto finish = std::chrono::high_resolution_clock::now();
   const double elapsedTime =
-      std::chrono::duration_cast<std::chrono::seconds>(finish - start).count();
+      std::chrono::duration_cast<std::chrono::duration<double>>(finish - start).count();

```

Comment:
This seems unrelated to this merge request. Was this a bug you found on the side?

Reply:
No, it is not a bug. Earlier, the time taken was printed in integers of seconds. So, for very small runs, the elapsed time showed zero seconds. Therefore I changed this to see the time elapsed in floating value in seconds.